### DISCUSSION TITLE
VTK output teaser

### DISCUSSION THREAD

**Discussion by slayoo** [open-atmos/PySDM#561] [Show and tell]
@atmos-cloud-sim-uj/pysdm-users 
Here is a teaser of what the VTK particle output for PSDM will enable us to obtain using, e.g., Paraview:

https://user-images.githubusercontent.com/89685/120945028-bcd2ea80-c737-11eb-8ba2-a1a685cd1449.mp4

Kudos to @abulenok for getting it up and running!
Stay tuned for a new release soon with many new features.

**Comment by trontrytel** [comment]
@slayoo and @abulenok This is really cool! Great progress
